packages/esm-patient-medications-app/src/components/medications-details-table.component.tsx

```diff
@@ -135,6 +135,9 @@ const MedicationsDetailsTable: React.FC<ActiveMedicationsProps> = ({
                 {formatDate(new Date(medication.dateStopped))}
               </span>
             ) : null}
+            {medication.dateStopped != null && !medication.hasOwnProperty('modified') && (
```

Comment:
@jwnasambu Where does the `medication.modified` property come from? That isn't part of the backend module and I don't see it being added in the frontend? `Discontinued` orders should have `action` property set to `DISCONTINUED` I think...

Reply:
@ibacher Kindly let me begin from here .

Reply:
@ibacher, after console logging the medication and checking the array properties, I didn't see anything like `modified` property. However, after careful consideration, I conclude that the endDate in this context is used to display medication details, specifically to show when a medication order was stopped, which suits both modified and discontinued statuses. Kindly I stand to be corrected.

Reply:
I don't know that it's guaranteed that `dateStopped` will always be set. That is, it will be set for medications that have been discontinued, but maybe not for those with a `autoStopDate` (sp. of property).